src/context/pertContext.tsx

```diff
@@ -45,9 +53,35 @@ const PertContextProvider: FC<Props> = ({ children }) => {
     round_to_next_minutes,
   };
 
+  const [isPertModalOpen, setIsPertModalOpen] = useState(false);
+  const [ticketNo, setTicketNo] = useState('');
   const [pertData, setPertData] = useState<IPertData>({ ...initialPertData });
 
-  const [isPertModalOpen, setIsPertModalOpen] = useState(false);
+  /**
+   * PERT-16:Estimation for each ticket will be stored in localStorage for a week.
+   * This gives the user the ability to re-estimate the ticket
+   */
+  useEffect(() => {
+    const list: pertListType[] | null = getPertStoredList();
```

Comment:
Its better if we can move the clearing out expired data part to App tsx from this context file :thinking: